fov_node:
  ros__parameters:
    fov_frame: map

    scan_topic: scan 
    deskewed_scan_topic: scan/deskewed 
    environment_scan_topic: scan/road_env 
    fov_topic: fov 
    environment_polygon_topic: visualization/road_env_polygon

    # do_correct_state_uncertainty: True
    # rotating_direction: CW
    # do_filter_scan: True
    # environment_boundary: [0., 2.5, 5., 2.5, 5., -3., 0., -3.]
    # subsample_rate: 10
    # view_range: 5.

    # error_models_directory: some/path/to/the/error/model
    # max_inclination_angle: 1.22173  # [rad]
    # # set these parameters in `ros_params_scenario.yaml`
    # # lidar error models z-values
    # apply_errors_models: True
    # range_stds_margin: 2.
    # angle_stds_margin: 2.
    # # fov state uncertainty z-values
    # CW convex polygon [x1, y1, x2, y2, ...] in the fov frame
    # state_pos_stds_margin: 2.
    # state_orient_stds_margin: 2.
